app/validators/index.js

```diff
@@ -59,7 +59,7 @@ export const validatePassword = (values, { t = i18nT() }) => {
         })
       },
       regex: {
-        pattern: '^((?=.*[0-9])(?=.*[A-Za-z])(?=.*[@$!%*#?&]))',
+        pattern: '^((?=.*[0-9])(?=.*[A-Za-z])(?=.*[~!@#$%^&_+=`|()[\\]{}":;\'\\/,.?/*#?&<>£-]))',
```

Comment:
Not sure if this fixes it, but according to regexpal anything after the unescaped `]` might be working in as expected. https://www.regexpal.com/

```suggestion
        pattern: '^((?=.*[0-9])(?=.*[A-Za-z])(?=.*[~!@#$%^&_+=`|()[\]{}":;\'\\/,.?/*#?&<>£-]))',
```